### PR TITLE
Weight and Product term fixes

### PR DESCRIPTION
**Pull request by nissimsan**

### CODE REVIEW

**Comment by nissimsan** [comment]
The signatures will conflict with https://github.com/w3c-ccg/traceability-vocab/pull/450

File: docs/openapi/components/schemas/common/TradeLineItem.yml [hunk 1]
```diff
@@ -70,17 +70,17 @@ properties:
     description: A measure of the net weight (mass).
     $ref: ./QuantitativeValue.yml
     $linkedData:
-      term: netWeight
+      term: tradeLineItemNetWeight
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/TradeLineItem.yml:73]: is there a reason this and other similar terms like `tradeLineItemGrossWeight` are prefixed by `tradeLineItem`?

Why isn't this just `netWeight` or `grossWeight`?

Comment (OR13) [reply docs/openapi/components/schemas/common/TradeLineItem.yml:73]: This should be fixed... prefxing is against the JSON-LD conventions we have.

**Review by OR13 — CHANGES_REQUESTED** [review]
Address term prefixing, by following the convention

File: docs/openapi/components/schemas/common/TradeLineItem.yml (hunk 1 above)
Comment (nissimsan) [reply docs/openapi/components/schemas/common/TradeLineItem.yml:73]: Agreed, @mkhraisha , let's stick to the no-prefix convention.

**Comment by nissimsan** [comment]
Agreement to merging once resolved

**Comment by nissimsan** [comment]
Naming convention done, conflict fixed. Merging as agreed.

**Pull request merged**